sk-usbhid.c

```diff
@@ -450,6 +450,15 @@ check_sk_options(fido_dev_t *dev, const char *opt, int *ret)
 		skdebug(__func__, "device is not fido2");
 		return 0;
 	}
+	/*
+	 * Workaround required up to libfido2 1.10.0.  As soon as 1.11.0
+	 * is released and updated in the Cygwin release, we can drop this.
+	 */
+	if (fido_dev_is_winhello(dev) && strcmp (opt, "uv") == 0) {
```

Comment:
does this hurt for libfido2 >= 1.11.0? would we need to pull it out later?

Reply:
It wouldn't hurt. But it should go away at one point, IMHO.